x-pack/test/functional/apps/ml/data_frame_analytics/outlier_detection_creation.ts

```diff
@@ -336,6 +364,42 @@ export default function ({ getService }: FtrProviderContext) {
           );
         });
 
+        it('adds discover custom url to the analytics job', async () => {
+          await ml.testExecution.logTestStep('opens edit flyout for discover url');
+          await ml.dataFrameAnalyticsTable.openEditFlyout(testData.jobId);
```

Comment:
Since the same 3 steps `openEditFlyout, openEditCustomUrlsForJobTab, and updateAnalyticsJob` are repeated, I think it would be better to add them to `addDiscoverCustomUrl`, `addDashboardCustomUrl`, and `addOtherTypeCustomUrl`.

Reply:
Combined in 19b1da5e11d89db1c7f5f817145fa537f03fc4ed